client/src/App.js

```diff
@@ -1,18 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState } from "react";
+import { Route } from "react-router-dom";
 
-import SavedList from './Movies/SavedList';
+import SavedList from "./Movies/SavedList";
+import MovieList from "./Movies/MovieList";
+import Movie from "./Movies/Movie";
+import MovieCard from "./Movies/MovieCard";
 
-const App = () => {
-  const [savedList, setSavedList] = useState( [] );
+const App = props => {
+  const [savedList, setSavedList] = useState([]);
 
   const addToSavedList = movie => {
-    setSavedList( [...savedList, movie] );
+    setSavedList([...savedList, movie]);
   };
 
   return (
     <div>
       <SavedList list={savedList} />
-      <div>Replace this Div with your Routes</div>
+      <Route exact path="/" component={MovieList} />
+      {/* <Route path="/movies/movie/:id" render={props => <Movie {...props} addToSavedList={addToSavedList} /> } /> */} 
+      <Route exact path="/movie/:id" component={Movie} />
+      {/* <Route  path="/movie/:id/moviecard" render={(props) => <MovieCard  data={props.match.params.id} />} /> */}
+      <MovieCard/>
```

Comment:
You're mounting the MovieCard component here, but I don't think that's needed for the scope of the project.  Furthermore, it's not being passed any props which is why it's not showing anything